### DISCUSSION TITLE
Modify tint2 script to run pavucontrol [pulseaudio]

### DISCUSSION THREAD

**Discussion by mrgreen3** [nwg-piotr/nwg-shell#307] [Ideas]
Currently I cannot find a volume control [that is in main arch repos] for controlling volume. I used to use volumeicon but it is alsa based. I came across your tint2 script that is again alsa based. 

Think I can modify it to work with pulseaudio, just wanted to know if that was ok?

Mr Green

**Comment by nwg-piotr** [comment]
Sure. I haven't touched these scripts since I moved to Wayland.
